packages/components/src/components/Slider/Slider.stories.tsx

```diff
@@ -0,0 +1,102 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type Meta, type StoryObj } from '@storybook/react';
+import { useRef, useState } from 'react';
+
+import { Slider } from './Slider';
+
+type Story = StoryObj<typeof Slider>;
+
+const meta: Meta<typeof Slider> = {
+    component: Slider,
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    args: {
+        'aria-labelledby': 'some-label-id',
+        'aria-label': 'Slider',
+    },
+};
+
+export default meta;
+
+export const SimpleSlider: Story = {};
+
+export const Disabled: Story = {
+    args: {
+        disabled: true,
+    },
+};
+
+export const MinimumAndMaximum: Story = {
+    args: {
+        min: 50,
+        max: 1000,
+    },
+};
+
+export const MinimumGap: Story = {
+    args: {
+        minStepsBetweenThumbs: 25,
+    },
+};
+
+export const StepSize: Story = {
+    args: {
+        step: 25,
+    },
+};
+
+export const Controlled: Story = {
```

Comment:
```suggestion
export const ExternallyControlled: Story = {
```

And move it to the bottom (so it matches the other stories)